pkg/edit/builtins_test.go

```diff
@@ -208,6 +208,71 @@ var bufferBuiltinsTests = []struct {
 		tk.CodeBuffer{Content: "ab", Dot: 1},
 		tk.CodeBuffer{Content: "a", Dot: 1},
 	},
+	{
+		"transpose-char",
```

Comment:
Give these test cases unique, meaningful names. This one can be called `transpose-char with dot at beginning`